app/src/main/res/layout/fragment_attendees.xml

```diff
@@ -1,40 +1,54 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.design.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    android:id="@+id/activity_attendee_list"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:orientation="vertical">
-    <RelativeLayout
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <data>
+        <import type="android.view.View" />
+        <import type="java.util.List" />
+        <variable
+            name="attendees"
+            type="List" />
+    </data>
+
+    <android.support.design.widget.CoordinatorLayout
         android:layout_width="match_parent"
         android:layout_height="match_parent">
 
         <android.support.v7.widget.RecyclerView
             android:id="@+id/rvAttendeeList"
             android:layout_width="match_parent"
             android:layout_height="match_parent"
-            android:paddingBottom="@dimen/image_small"
```

Comment:
this padding was **bottom padding of a last item in the list**. I am talking about this effect - 

Without this padding last item will be overlapped by floating action button.

![img_20170619_115423_173](https://user-images.githubusercontent.com/13533840/27272164-6edb925e-54e6-11e7-93dc-369e89593174.jpg)

With the padding it won't

![img_20170619_115442_492](https://user-images.githubusercontent.com/13533840/27272174-7d2b00ce-54e6-11e7-85e4-6aca0bb5dac6.jpg)